src/rules/enforce-fieldpath-syntax-in-docsetter.ts

```
@@ -0,0 +1,239 @@
+        if (
+          property.key.type === AST_NODE_TYPES.Literal &&
+          typeof property.key.value === 'string' &&
+          property.key.value.includes('.')
+        ) {
+          continue;
+        }
+
+        // Check if the value is an object (but not an array)
+        if (value.type === AST_NODE_TYPES.ObjectExpression) {
+          // Skip nested objects that contain spread elements or computed properties
+          const hasSpreadOrComputed = value.properties.some(prop =>
```

ESLint

[error] Replace `prop` with `⏎············(prop)`

(prettier/prettier)